fdbclient/NativeAPI.actor.cpp

```diff
@@ -255,16 +255,27 @@ void DatabaseContext::getLatestCommitVersions(const Reference<LocationInfo>& loc
 
 	std::map<Version, std::set<Tag>> versionMap; // order the versions to be returned
 	for (int i = 0; i < locationInfo->locations()->size(); i++) {
+		int updatedVersionMap = 0;
+		Version commitVersion = invalidVersion;
+		Tag tag = invalidTag;
 		UID uid = locationInfo->locations()->getId(i);
 		if (ssidTagMapping.find(uid) != ssidTagMapping.end()) {
-			Tag tag = ssidTagMapping[uid];
+			tag = ssidTagMapping[uid];
 			if (ssVersionVectorCache.hasVersion(tag)) {
-				Version commitVersion = ssVersionVectorCache.getVersion(tag); // latest commit version
+				commitVersion = ssVersionVectorCache.getVersion(tag); // latest commit version
 				if (commitVersion < readVersion) {
 					versionMap[commitVersion].insert(tag);
 				}
 			}
 		}
+		if (!updatedVersionMap) {
```

Comment:
Looks like `updatedVersionMap` is only initialized, not updated.

Reply:
I must have had a typo while preparing the fix for review. Thanks !